src/Core/ContentType/Transformer/BaseHtmlTransformer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Core\ContentType\Transformer;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+abstract class BaseHtmlTransformer extends AbstractTransformer
+{
+    protected function setTransformed(TransformContext $context, Crawler $crawler): void
+    {
+        $data = $context->getData();
+        $transformed = $crawler->outerHtml();
+
+        if (false === \strpos($data, '<html>')) {
+            $transformed = \str_replace(['<html>', '</html>'], '', $transformed);
+        }
+        if (false === \strpos($data, '<body>')) {
+            $transformed = \str_replace(['<body>', '</body>'], '', $transformed);
+        }
```

Comment:
What about html or body tag with attributes?

Reply:
Very good remark:

https://github.com/ems-project/EMSCoreBundle/pull/722/commits/856a516730f439cd9222b7801f0c8cda007a7961

Fixed and even changed the test case